controllers/members.js

```diff
@@ -83,11 +84,22 @@ const archiveMembers = async (req, res) => {
   try {
     const { username } = req.params;
     const user = await dataAccess.retrieveUsers({ username });
+    const superUserId = req.userData.id;
```

Comment:
are we sure here that `req.userData.id` will give here super user id??

Reply:
yes, if the user has access to put the users to achieve, they are authorized user